packages/vanilla-js/src/AuthNFlowClient/types.ts

```diff
@@ -37,6 +38,7 @@ export namespace AuthFlow {
     AGREEMENTS = "agreements",
     PROFILE = "profile",
     PROVISIONAL = "provisional_enrollment",
+    COMPLETE = "complate",
```

Comment:
"complate" typo?

Reply:
Thanks for catching that, I'll update in my next PR. It won't break anything.